docs/tutorial/offscreen-rendering.md

```diff
@@ -1,59 +1,66 @@
 # Offscreen Rendering
 
-Offscreen rendering lets you obtain the content of a browser window in a bitmap,
-so it can be rendered anywhere, for example on a texture in a 3D scene. The
-offscreen rendering in Electron uses a similar approach than the [Chromium
+## Overview
+
+Offscreen rendering lets you obtain the content of a browser window in a
+bitmap, so it can be rendered anywhere, for example, on texture in a 3D scene.
+The offscreen rendering in Electron uses a similar approach than the [Chromium
 Embedded Framework](https://bitbucket.org/chromiumembedded/cef) project.
 
-Two modes of rendering can be used and only the dirty area is passed in the
-`'paint'` event to be more efficient. The rendering can be stopped, continued
-and the frame rate can be set. The specified frame rate is a top limit value,
-when there is nothing happening on a webpage, no frames are generated. The
-maximum frame rate is 60, because above that there is no benefit, only
-performance loss.
+There are two rendering modes that can be used (see the section below) and only
+the dirty area is passed to the `paint` event to be more efficient. You can
+stop/continue the rendering as well as set the frame rate. The specified frame
+rate is a top limit value, when nothing happening on a webpage, no frames are
+generated. The maximum frame rate is 60 because greater values bring only
+performance losses with no benefits.
 
-**Note:** An offscreen window is always created as a [Frameless Window](../api/frameless-window.md).
+An offscreen window is always created as a
+[Frameless Window](../api/frameless-window.md).
```

Comment:
Can we turn these sentences into notes rather than paragraphs?

Something like

**Notes**:
* Note 1
* Note 2